packages/cf-style-provider/src/createRenderer.js

```diff
@@ -47,6 +48,7 @@ const createRenderer = opts => {
     enhancers,
     selectorPrefix: usedOpts.selectorPrefix,
     mediaQueryOrder: [
```

Comment:
It theory. But objects don't have to preserve the order of properties so it's not safe. That's why that API is using an array in the first place.